apps/src/templates/teacherDashboard/teacherSectionsReduxSelectors.js

```diff
@@ -164,6 +164,15 @@ export const sectionFromServerSection = serverSection => ({
   courseOfferingId: serverSection.course_offering_id,
   courseVersionId: serverSection.course_version_id,
   courseDisplayName: serverSection.course_display_name,
+  course: serverSection.course
+    ? {
+        courseOfferingId: serverSection.course.course_offering_id,
+        versionId: serverSection.course.version_id,
+        unitId: serverSection.course.unit_id,
+        hasLessonExtras: serverSection.course.lesson_extras_available,
```

Comment:
I decided to stick with the names in the components... open to pushback on that choice.

Reply:
Normally I would like this, but I want to stay away from having two different styles in this already complex component. I think we need to have everything migrated over to camel case or nothing.